app/views/compute_resources_vms/form/ovirt/_network.html.erb

```diff
@@ -1,5 +1,7 @@
 <%= text_f f, :name, :disabled => !new_vm, :class => "ovirt_name", :label => _('Name'), :label_size => "col-md-3" %>
-<% selected_cluster ||= compute_resource.clusters.first.id %>
+
+<% selected_cluster ||= params[:host] && params[:host][:compute_attributes] && params[:host][:compute_attributes][:cluster]  %>
+<% selected_cluster ||= compute_resource.clusters.first.id  %>
```

Comment:
I know you haven't written this code, but perhaps changing this to be more defensive if no cluster is found would be better to avoid 500, how about `compute_resource.clusters.first.try(:id)`